.github/workflows/docs.yml

```diff
@@ -0,0 +1,35 @@
+name: Documentation
```

Comment:
```suggestion
# This is a GitHub workflow defining a set of jobs with a set of steps.
# ref: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
#
name: Documentation
```

I love having easy access to the reference, and you may need it now that you read up on the `permissions` field perhaps.

Reply:
Besides the change above, perhaps also adding a note to clarify that this doucmentation is published on gh-pages directly, and we are not relying on readthedocs or similar.